Cmake on travis is failing due to:
```
CMake Error at config/FindIconv.cmake:63 (cmake_push_check_state):
  Unknown CMake command "cmake_push_check_state".
Call Stack (most recent call first):
  config/findDependencies.cmake:53 (include)
  CMakeLists.txt:88 (include)
```
I guess the `FindIconv.cmake` uses too recent cmake features for cmake on travis. This might become a problem for the CI jobs on GitLab, as Debian tends to lack behind a bit (surprisingly CentOS ships cmake 3.11)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 # CMakeLists.txt for exiv2 library
 
-cmake_minimum_required( VERSION 3.1.0 )
+cmake_minimum_required( VERSION 3.3.2 )
 project( exiv2 )
 
 include(GNUInstallDirs)

--- a/config/FindIconv.cmake
+++ b/config/FindIconv.cmake
@@ -60,7 +60,7 @@ endif()
 # If any cache variable is already set, we'll skip this test.
 if(NOT DEFINED Iconv_IS_BUILT_IN)
   if(UNIX AND NOT DEFINED Iconv_INCLUDE_DIR AND NOT DEFINED Iconv_LIBRARY)
-    cmake_push_check_state(RESET)
+    include(CheckCSourceCompiles)
     # We always suppress the message here: Otherwise on supported systems
     # not having iconv in their C library (e.g. those using libiconv)
     # would always display a confusing "Looking for iconv - not found" message
@@ -87,7 +87,6 @@ if(NOT DEFINED Iconv_IS_BUILT_IN)
     else()
       check_cxx_source_compiles("${Iconv_IMPLICIT_TEST_CODE}" Iconv_IS_BUILT_IN)
     endif()
-    cmake_pop_check_state()
   else()
     set(Iconv_IS_BUILT_IN FALSE)
   endif()

--- a/config/findDependencies.cmake
+++ b/config/findDependencies.cmake
@@ -50,7 +50,7 @@ if( EXIV2_ENABLE_NLS )
     # the manual check in config/generateConfigFile.cmake
 endif( )
 
-include( FindIconv )
+find_package(Iconv)
 if( ICONV_FOUND )
     message ( "-- ICONV_INCLUDE_DIR : " ${Iconv_INCLUDE_DIR} )
     message ( "-- ICONV_LIBRARIES : " ${Iconv_LIBRARY} )

